packages/lib/payment/handleNoShowFee.ts

```diff
@@ -69,23 +69,19 @@ export const handleNoShowFee = async ({
     throw new Error("User ID is required");
   }
 
-  const attendeesListPromises = [];
+  const attendee = booking.attendees[0];
 
-  for (const attendee of booking.attendees) {
-    const attendeeObject = {
+  const attendeesList = [
+    {
       name: attendee.name,
       email: attendee.email,
       timeZone: attendee.timeZone,
       language: {
         translate: await getTranslation(attendee.locale ?? "en", "common"),
```

Comment:
We only need to getTranslations for the first attendee of the booking so we're removing the looped `await`